### PR TITLE
Harisdev netizen patch 1

### PR DESCRIPTION
**Pull request by harisdev-netizen**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6072"></a>

### CODE REVIEW

File: data/harisdev.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Computer Science guy in a Software Engineering world!",
+    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png"
```
Comment (Panquesito7) [inline data/harisdev.json:6]: The `avatar` field is not required. Your avatar is automatically fetched from your GitHub profile.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Your filename must match your GitHub username.
Please update the filename accordingly.

File: data/harisdev.json (hunk 1 above)
Comment (harisdev-netizen) [reply data/harisdev.json]: Done

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/6072/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](http://discord.eddiehub.org). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

File: data/harisdev-netizen.json [hunk 2]
```diff
@@ -1,10 +1,12 @@
 {
-  "name": "Haris Khan",
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Computer Science guy in a Software Engineering world!",
+    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png"
```
Comment (Panquesito7) [inline data/harisdev-netizen.json:6]: The avatar is automatically fetched from your GitHub profile.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/harisdev-netizen.json [hunk 3]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "Haris Khan",
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/harisdev-netizen.json:3]: This is no longer required
```suggestion
```

File: data/harisdev-netizen.json [hunk 4]
```diff
@@ -17,28 +17,27 @@
...
-    {
-      "name": "GitHub: Checkout my Projects",
-      "url": "https://github.com/harisdev-netizen",
-      "icon": "FaGithub"
-    },
-    {
-      "name": "Hashnode: Read my Blogs",
-      "url": "https://harisdev.hashnode.dev",
-      "icon": "SiHashnode"
     },
```
Comment (SaraJaoude) [inline data/harisdev-netizen.json:20]: comma not required
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**